Source: written by Nick Hahn <nick.hahn@hotmail.de>
Upstream: pending
Reason: nextcloud-desktop fails during compilation because it does not support LibreSLL
natively

--- src/libsync/clientsideencryption.cpp.orig	2018-08-24 14:54:42.000000000 +0200
+++ src/libsync/clientsideencryption.cpp	2018-10-22 22:00:10.099275292 +0200
@@ -510,7 +511,7 @@
         return {};
     }
 
-    if (EVP_PKEY_CTX_set_rsa_oaep_md(ctx, EVP_sha256()) <= 0) {
+    if (EVP_PKEY_CTX_set_rsa_mgf1_md(ctx, EVP_sha256()) <= 0) {
         qCInfo(lcCseDecryption()) << "Error setting OAEP SHA 256";
         handleErrors();
         return {};
@@ -573,7 +574,7 @@
         exit(1);
     }
 
-    if (EVP_PKEY_CTX_set_rsa_oaep_md(ctx, EVP_sha256()) <= 0) {
+    if (EVP_PKEY_CTX_set_rsa_mgf1_md(ctx, EVP_sha256()) <= 0) {
         qCInfo(lcCse()) << "Error setting OAEP SHA 256";
         exit(1);
     }

